tools/extractgzlist

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 
-release="precise"
+release="jammy"
```

Comment:
Can confirm these changes are necessary to even download.
`precise` manpages e.g. https://manpages.ubuntu.com/manpages/precise/en/man8 no longer available, they all return 404.

And any http: now redirects to https: (could work with `curl --location` but startign from https: is of course more secure).

The other changes are just for rendered links to be correct :+1:

Specifically, the steps to download:
1. `tools/extractgzlist`
2. Edit `tools/extractgzlist` to `section="1"` and run it again.
3. Edit `tools/dlgzlist` to end with `done < gzlist8` and run it (took ~4hours)
4. Edit `tools/dlgzlist` to end with `done < gzlist1` and run it. Will take over a day, I reduced the sleep range to `(500, 1000)` and it still worked, took ~16hours.